Sources/ApiVideoPlayer/ApiVideoPlayerController.swift

```diff
@@ -387,6 +391,10 @@ public class ApiVideoPlayerController: NSObject {
         return nil
     }
 
+    public var currentSpeedRate: Double? {
```

Comment:
Is there a reason why it is a double whereas AVPlayer uses a Float? (which make usage a Double useless)